assume/strategies/naive_strategies.py

```diff
@@ -137,6 +137,83 @@ def calculate_bids(
         bids = [order]
 
         bids = self.remove_empty_bids(bids)
+        return bids
+
+
+class NaiveDASteelplantStrategy(BaseStrategy):
+    def calculate_bids(
+        self,
+        unit: SupportsMinMax,
+        market_config: MarketConfig,
+        product_tuples: list[Product],
+        **kwargs,
+    ) -> Orderbook:
+        bids = []
+        start = product_tuples[0][0]  # start time of the first product
+
+        if unit.opt_power_requirement is None:
+            if unit.objective == "min_variable_cost":
+                unit.determine_optimal_operation_without_flex()
+
+        # unit.run_modified_optimization()
+
+        bids = []
+        for product in product_tuples:
+            """
+            for each product, calculate the marginal cost of the unit at the start time of the product
+            and the volume of the product. Dispatch the order to the market.
+            """
+            start = product[0]
+            volume = unit.opt_power_requirement.loc[start]
+            price = 3000
+            bids.append(
+                {
+                    "start_time": product[0],
+                    "end_time": product[1],
+                    "only_hours": product[2],
+                    "price": price,
+                    "volume": -volume,
+                }
+            )
+
+        return bids
```

Comment:
The two strategies are basically the same as the NaiveSingleBidStrategy
but an additional method of the unit is called and the result is retrieved from `unit.opt_power_requirement` instead of unit.outputs["energy"].

If we would find a different way to call the calculation/optimization model before bidding, we would not need another strategy, right?
Just for my understanding.

Reply:
Two different stategies are created keeping in mind distinguising logic of determining bid price for day ahead and redispatch , each.